src/clj/test/cards/operations.clj

```diff
@@ -1358,3 +1358,19 @@
       (prompt-select :corp (refresh vanilla))
       (is (not= "Wetwork Refit" (:title (first (:hosted (refresh vanilla)))))
           "Wetwork Refit is not hosted on Vanilla"))))
+
+(deftest offer-you-cant-refuse
+  ;; An Offer You Can't Refuse - exact card added to score area, not the last discarded one
+  (do-game
+    (new-game (default-corp [(qty "Celebrity Gift" 1) (qty "An Offer You Can't Refuse" 1)])
+              (default-runner))
+    (play-from-hand state :corp "An Offer You Can't Refuse")
+    (prompt-choice :corp "R&D")
+    (core/move state :corp (find-card "Celebrity Gift" (:hand (get-corp))) :discard)
+    (is (= 2 (count (:discard (get-corp)))))
+    (prompt-choice :runner "No")
+    (is (= 1 (:agenda-point (get-corp))) "An Offer the Runner refused")
+    (is (= 1 (count (:scored (get-corp)))))
+    (is (= "An Offer You Can't Refuse" (:title (first (:scored (get-corp))))))
```

Comment:
A nice utility in the test framework is `find-card`, which takes a title and collection and returns the card with that title if it is in the colletion, `nil` otherwise. You can use this with `is` pretty nicely:

`(is (find-card "An Offer You Can't Refuse" (:scored (get-corp))) ...)`

which will fail if the card can't be found.